.github/workflows/deploy_to_firebase_app_distribution.yml

```diff
@@ -13,10 +13,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
+          distribution: 'temurin'
```

Comment:
Can you please explain why do we need to add this configuration? 🤔

Reply:
@hoangnguyen92dn Gradle 7 requires Java 11 to run 🤓 https://github.com/nimblehq/jetpack-compose-crypto/runs/8184091690?check_suite_focus=true. Let me add this info to PR description.